packages/graphics/src/GraphicsGeometry.js

```diff
@@ -599,6 +599,8 @@ export default class GraphicsGeometry extends BatchGeometry
 
         currentGroup.textureCount = 0;
         currentGroup.start = 0;
+        currentGroup.size = 0;
+        currentGroup.type = 4;
```

Comment:
It took me a while to discover this is actually `WebGLRenderingContext.TRIANGLES`. Can we use the real constants, or at least named constants that proxy the values? I see that the values `1` and `4` are used in a few places without names too. The type in BatchDrawCall isn't documented either which made this very difficult to discover.

Reply:
Good point 👍

Reply:
It was like that before I edited it. OK, fixed :)